### ISSUE TITLE
`AddressParseError`doesn't implement `std::fmt::Display`

### ISSUE THREAD

**Issue by jgraef** [nimiq/core-rs#26]
I ran into this multiple times, but never got the time to fix it. So here is the issue open for anyone to fix. Or I will fix it, once I got some spare time.

```
error[E0277]: `keys::AddressParseError` doesn't implement `std::fmt::Display`
  --> src/config.rs:52:53
   |
52 |     Address::from_user_friendly_address(&s).map_err(Error::custom)
   |                                                     ^^^^^^^^^^^^^ `keys::AddressParseError` cannot be formatted with the default formatter
   |
   = help: the trait `std::fmt::Display` is not implemented for `keys::AddressParseError`
   = note: in format strings you may be able to use `{:?}` (or {:#?} for pretty-print) instead
   = note: required by `config::_IMPL_DESERIALIZE_FOR_GenesisConfig::_serde::de::Error::custom`
```

Since `keys::AddressParseError` is an error, we should also implement the necessary traits.

**Comment by jgraef** [comment]
The error type `keys::AddressParseError` is an enum. We already use `enum_display_derive` at some places. So this would be an easy fix.

**Comment by jgraef** [comment]
Fixed with commit [`e596e79ac70815055f5a068143ab5dabb15a63e7`](https://github.com/nimiq/core-rs/commit/e596e79ac70815055f5a068143ab5dabb15a63e7)
